seed/views/v3/compliance_metrics.py

```diff
@@ -80,7 +80,7 @@ def retrieve(self, request, pk=0):
     @require_organization_id_class
     @api_endpoint_class
     @ajax_request_class
-    @has_perm_class('requires_member')
+    @has_perm_class('requires_root_member_access')
```

Comment:
make sure to socialize this with the team, that is the new `requires_root_member_access` perm class.

Reply:
👍